plugins/public/cinq-collector-aws/cinq_collector_aws/account.py

```diff
@@ -105,31 +105,36 @@ def update_rds_databases(self):
             response = lambda_client.invoke(FunctionName=self.rds_function_name, InvocationType='RequestResponse',
                                             Payload=input_payload
                                             )
+            print(response)
             rds_dbs = json.loads(response['Payload'].read().decode('utf-8'))
-
-            for db_instance in rds_dbs:
-                metrics = None
-                if 'metrics' in db_instance.keys():
-                    metrics = db_instance['metrics']
-                tags = {t['Key']: t['Value'] for t in db_instance['tags'] or {}}
-                properties = {
-                    'tags': tags,
-                    'metrics': metrics,
-                    'creation_date': datetime.utcnow()
-                }
-                if db_instance['resource_name'] in existing_rds_dbs:
-                    rds = existing_rds_dbs[db_instance['resource_name']]
-                    if rds.update(db_instance, properties):
-                        self.log.debug('Change detected for RDS instance {}/{} '
-                                       .format(db_instance['resource_name'], properties))
-                else:
-                    RDSInstance.create(
-                        db_instance['resource_name'],
-                        account_id=self.account.account_id,
-                        location=db_instance['location'],
-                        properties=properties,
-                        tags=tags
-                    )
+            if 'errorMessage' in rds_dbs:
+                self.log.error('RDS Lambda Execution Failed / {}'.format(rds_dbs['errorMessage']))
+                raise
+
+            if rds_dbs:
+                for db_instance in rds_dbs:
+                    metrics = None
+                    if 'metrics' in db_instance.keys():
+                        metrics = db_instance['metrics']
+                    tags = {t['Key']: t['Value'] for t in db_instance['tags'] or {}}
+                    properties = {
+                        'tags': tags,
```

Comment:
Can we pass the DB type in? (e.g. MySQL, Aurora, etc.)
The Action taker need this property

Reply:
ok will add